src/main/java/terry/entity/ToDo.java

```diff
@@ -0,0 +1,84 @@
+package terry.entity;
+
+import org.json.JSONObject;
+import terry.cmd.CmdOptArg;
+import terry.exception.MissingOptArgException;
+
+import java.util.List;
+
+/**
+ * Stores todos.
+ */
+public class ToDo {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Checks if a list of {@link CmdOptArg} has enough number of options.
+     * @exception MissingOptArgException when parameter count is less than
+     * {@code leastCount}
+     * */
+    protected static void checkArgCount(List<CmdOptArg> optArgList, int leastCount) throws MissingOptArgException {
```

Comment:
while not particularly long, this line is probably longer than most others in the codebase.